tests/console/yast2_bootloader.pm

```diff
@@ -27,6 +26,7 @@ sub run() {
     script_run("/sbin/yast2 bootloader; echo yast2-bootloader-status-\$? > /dev/$serialdev", 0);
     assert_screen "test-yast2_bootloader-1", 300;
     send_key "alt-o";                                     # OK => Close
+    if (assert_screen 'yast2_bootloader-missing_package') { send_key 'alt-i'; }
```

Comment:
we got rid of these needle checks for finished yast modules on consoles for a reason. Don't reinvent them.